code/script.js

```diff
@@ -222,86 +223,148 @@ const generateBoard = () => {
+   if (category === 'hair') {
     if (keep) {
-      alert(
-        `Yes, the person wears ${value}! Keep all people that wears ${value}`
-      )
-    } else {
-      alert(
-        `No, the person doesn't wear ${value}! Remove all people that wears ${value}`
-      )
+      alert(`Oh yeah, the person has ${value} hair! Keep all people that has ${value} hair.`)
+      charactersInPlay = charactersInPlay.filter((person) => person[category] === value) //Attribute changed to category
     }
-  } else if (category === 'other') {
-    // Similar to the one above
-  } else {
-    if (keep) {
-      // alert popup that says something like: "Yes, the person has yellow hair! Keep all people with yellow hair"
-    } else {
-      // alert popup that says something like: "No, the person doesnt have yellow hair! Remove all people with yellow hair"
+    else {
+      alert(`Oh no, the person doesn’t have ${value} hair! Remove all people that have ${value} hair.`)
+      charactersInPlay = charactersInPlay.filter((person) => person[category] !== value)
     }
-  }
 
-  // Determine what is the category
-  // filter by category to keep or remove based on the keep variable.
-  /* 
-    for hair and eyes :
-      charactersInPlay = charactersInPlay.filter((person) => person[attribute] === value)
-      or
-      charactersInPlay = charactersInPlay.filter((person) => person[attribute] !== value)
-
-    for accessories and other
+    if (category === 'eyes') {
+      if (keep) {
+        alert(`Wohoo, the person has ${value} eyes! Keep all people that has ${value} eyes.`)
+        charactersInPlay = charactersInPlay.filter((person) => person[category] === value) 
+      }
+      else {
+        alert(`Nope, the person doesn’t have ${value} eyes! Remove all people that have ${value} eyes.`)
+        charactersInPlay = charactersInPlay.filter((person) => person[category] !== value)
+      }
+    }
+  
+   if (category === 'accessories') {
+    if (keep) {
+      alert(`Yes, the person wears ${value}! Keep all people that wears ${value}`)
       charactersInPlay = charactersInPlay.filter((person) => person[category].includes(value))
-      or
+    } 
+    else {
+      alert(`No, the person doesn't wear ${value}! Remove all people that wears ${value}`)
       charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value))
-  */
+    }
 
-  // Invoke a function to redraw the board with the remaining people.
+  if (category === 'other') {
+    if (keep) {
+      alert(`Yes, the person ${value}! Keep all people that is ${value}`)
+      charactersInPlay = charactersInPlay.filter((person) => person[category].includes(value))
+    } 
+    else {
+      alert(`No, the person doesn't ${value}! Remove all people that is ${value}`)
+      charactersInPlay = charactersInPlay.filter((person) => !person[category].includes(value))
+    }
+    console.log('test five filteroutwithalerts')
+  }
+}
 }
+}
+
+generateBoard()
```

Comment:
This have ended up at the wrong side of the curly bracket, it should be before the one at line-365. Probably one of the things that made the board stop working, you now have two commands that say the same thing but at different places. When they clash, nothing works.